### PR TITLE
#1381 - Team page

### PR DESCRIPTION
**Pull request by rajzshkr**
Doubts:-
- [x] Link placement in the header.
- [x] Contributors image src from twitter or GitHub.
- [x] Any updates need for Design?

### CODE REVIEW

**Comment by hzoo** [comment]
Looks pretty nice! I guess the github picture is better since maybe not everyone has twitter, but they should have a github?

File: _data/team.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    twitter: amasad
+  - name: James Kyle
+    github: thejameskyle
+    twitter: thejameskyle
+  - name: Jesse McCarthy
+    github: jmm
+    twitter: mccjm
+  - name: Sebastian McKenzie
+    github: sebmck
+    twitter: kittens
```
Comment (xtuc) [inline _data/team.yml]: It's twitter: sebmck and github: kittens

**Comment by existentialism** [comment]
Nice, small alignment nit:

image

File: _data/team.yml (hunk 1 above)
Comment (rajzshkr) [reply _data/team.yml]: Done

**Comment by rajzshkr** [comment]
Ping @existentialism @xtuc

**Comment by xtuc** [comment]
lgtm.

I'm just wondering, it would be so cool to include all the contributers from github. Since we're already fetching the READMEs from GitHub, it seems not so difficult. What do you guys think? Of couse we can merge this first.

**Comment by rajzshkr** [comment]
@xtuc we can do that.
@hzoo  I've created a [pull request](https://github.com/babel/babel/pull/6541) in babel/babel with the updated link

**Comment by existentialism** [comment]
@rajzshkr great work!

re: including all contributors, maybe as a followup task, we can have a section below the team that is just like 30x30 avatars only with a link to their github?

File: team.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          <p>{% include icons/github.svg %}<a href="//www.github.com/{{member.github}}" target="_blank">{{member.github}}</a></p>
+          <p>{% include icons/twitter.svg %}<a href="//www.twitter.com/{{member.twitter}}" target="_blank">{{member.twitter}}</a></p>
+      </div>
+    {% endfor %}
+  </div>
+
+  <div class="row">
+    <h3 class="team__name">Members</h3>
+    {% for member in site.data.team.members %}
+      <div class="col-md-3 col-sm-4 col-xs-12 team__member">
```
Comment (existentialism) [inline team.md]: Just a thought, since all these blocks are the same, we could add a jekyll include?

```erb
{% include member.html member=member %}
```

Comment (rajzshkr) [reply team.md]: Done.

Comment (rajzshkr) [reply team.md]: We can track showing all contributor changes in team page in [this issue](https://github.com/babel/website/issues/1431)

**Comment by existentialism** [comment]
@rajzshkr thanks again!

**Pull request merged**